projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid.spec.ts

```diff
@@ -1713,6 +1714,42 @@ describe('Basic IgxHierarchicalGrid #hGrid', () => {
         }));
     });
 
+    describe('IgxHierarchicalGrid Multi-Column Headers', () => {
+        let fixture: ComponentFixture<IgxHierarchicalGridMCHComponent>;
+        let hierarchicalGrid: IgxHierarchicalGridComponent;
+
+        beforeEach(() => {
+            fixture = TestBed.createComponent(IgxHierarchicalGridMCHComponent);
+            fixture.detectChanges();
+            hierarchicalGrid = fixture.componentInstance.hGrid;
+        });
+
+        it('should fire expandedChange, hiddenChange and pinnedChange events for child grid.', () => {
+            const childGrid = hierarchicalGrid.gridAPI.getChildGrids(false)[0];
+            const columnGroup2 = childGrid.columns[4] as IgxColumnGroupComponent;
+            const columnGroup2Header = GridFunctions.getColumnGroupHeaders(fixture)[3];
+            const expandIcon = columnGroup2Header.queryAll(By.css('.igx-icon'))[0];
+            const pinIcon = columnGroup2Header.queryAll(By.css('.igx-icon'))[1];
+
+            expect(columnGroup2.expanded).toBeFalsy();
```

Comment:
Both properties are boolean. You should not be using Falsy/Truthy for boolean properties. You can use `toBeTrue()`